docs/Introduction/getting-started/conceptual-overview.md

```diff
@@ -0,0 +1,176 @@
+excerpt: "Smart Contracts and Chainlink"
+whatsnext: {"Deploy Your First Smart Contract":"/docs/deploy-your-first-contract/", "Consuming Data Feeds":"/docs/consuming-data-feeds/"}
+metadata:
+  title: "Conceptual Overview"
+  description: "Learn the basic concepts about what smart contracts are and, how to write them, and how Chainlink oracles work with smart contracts."
+  image:
+    0: "/files/1a63254-link.png"
+---
+
+Welcome to the Smart Contract Getting Started guide. This overview explains the basic concepts of smart contract development and oracle networks.
+
+**Skip ahead**
+To get your hands on the code right away, you can skip this overview:
+- If you are new to smart contracts, [Deploy Your First Smart Contract](/docs/deploy-your-first-contract/) in an interactive web development environment.
+- If you are already familiar with smart contracts and want to learn how to create *hybrid* smart contracts that use Chainlink Data Feeds, skip to the [Consuming Data Feeds](/docs/consuming-data-feeds/) guide.
+
+<p>
+  https://www.youtube.com/watch?v=rFXSEEQG9YE
+</p>
+
+## Table of Contents
+
++ [What is a smart contract? What is a hybrid smart contract?](#what-is-a-smart-contract-what-is-a-hybrid-smart-contract)
++ [What language is a smart contract written in?](#what-language-is-a-smart-contract-written-in)
++ [What does a smart contract look like?](#what-does-a-smart-contract-look-like)
++ [What does "deploying" mean?](#what-does-deploying-mean)
++ [What are oracles? Why are they important?](#what-are-oracles)
++ [How do smart contracts use oracles?](#how-do-smart-contracts-use-oracles)
++ [What is Remix?](#what-is-remix)
++ [What is Metamask?](#what-is-metamask)
+
+## What is a smart contract? What is a hybrid smart contract?
+
+When deployed to a blockchain, a *smart contract* is a set of instructions that can be executed without intervention from third parties. The code of a smart contract determines how it responds to input, just like the code of any other computer program.
+
+A valuable feature of smart contracts is that they can store and manage on-chain assets (like [ETH or ERC20 tokens](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/)), just like you can with an Ethereum wallet. Because they have an on-chain address like a wallet, they can do everything any other address can. This opens the door for programming automated actions when receiving and transferring assets.
+
+Smart contracts can connect to real-world market prices of assets to produce powerful applications. Being able to securely connect smart contracts with off-chain data and services is what makes them *hybrid* smart contracts. This is done using oracles.
+
+## What language is a smart contract written in?
+
+The most popular language for writing smart contracts on Ethereum is [Solidity](https://docs.soliditylang.org/en/v0.8.7/). It was created by the Ethereum Foundation specifically for smart contract development and is constantly being updated.
+
```

Comment:
I feel like we need to at least mention Rust/Solana/others here.

Reply:
I added the non-EVM chains to this section with their supported languages; links to their language documentation.